fhir-ig-davinci-pdex-plan-net/src/main/java/com/ibm/fhir/ig/davinci/pdex/plan/net/util/ResourceProcessor.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.davinci.pdex.plan.net.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonReader;
+import javax.json.JsonReaderFactory;
+import javax.json.JsonWriter;
+import javax.json.JsonWriterFactory;
+
+/**
+ * This class fixes two issues with the packaged DaVinci PDEX Plan Net artifacts:
+ * <ol>
+ * <li>The provided structure definitions contain invalid XHTML</li>
+ * <li>The provided structure definitions do not contain a version element</li>
+ * </ol>
+ */
+public class ResourceProcessor {
```

Comment:
I know its not a test, but maybe we should move it there so that its not included in the actual Jar

Reply:
If we update the artifacts in the future, then I think it makes sense to keep it in the main src folder tree because we will run it again and it seems more like main code than test code in that regard.